File: src/main/java/fi/aalto/cs/apluscourses/intellij/notifications/ApiTokenNotSetNotification.java
package fi.aalto.cs.apluscourses.intellij.notifications;

import static fi.aalto.cs.apluscourses.utils.PluginResourceBundle.getText;

import com.intellij.notification.Notification;
import com.intellij.notification.NotificationType;
import fi.aalto.cs.apluscourses.services.PluginSettings;

public class ApiTokenNotSetNotification extends Notification {
  /**
   * Constructs a notification that tells authentication is not set.
   */
  public ApiTokenNotSetNotification() {
    super(
        PluginSettings.A_PLUS,
        getText("notification.ApiTokenNotSet.title"),
        getText("notification.ApiTokenNotSet.content"),
        NotificationType.INFORMATION);
  }
}

File: src/main/kotlin/fi/aalto/cs/apluscourses/notifications/ApiTokenNotSetNotification.kt
package fi.aalto.cs.apluscourses.notifications

import com.intellij.notification.Notification
import com.intellij.notification.NotificationType
import fi.aalto.cs.apluscourses.MyBundle
import fi.aalto.cs.apluscourses.services.PluginSettings

class ApiTokenNotSetNotification
/**
 * Constructs a notification that tells authentication is not set.
 */
    : Notification(
    PluginSettings.A_PLUS,
    MyBundle.message("notification.ApiTokenNotSet.title"),
    MyBundle.message("notification.ApiTokenNotSet.content"),
    NotificationType.INFORMATION
)
